### PR TITLE
Bump Psammead components and use new fontface

### LINKED ISSUE #1276: Display italicised text in headlines & subheadings
**Is your feature request related to a problem? Please describe.**
Blocked on and https://github.com/bbc/simorgh/issues/1356 May also require further work on Psammead-headings -- i.e. a CSS rule added so child `<i>` tags use the correct font-family? 
Headlines & subheadlines should respect the "emphasis" text fragment attribute.

**Describe the solution you'd like**
When an article has a headline or subheading containing text marked with the "emphasis", we will display the appropriate Reith italics font, with the string contained inside an `<i>` tag, for semantic information.

Currently we will ignore any other rich-text option that appear in headlines or subheadings.

If not handled by #1356, we will need to be able to conditionally render certain fragment attributes.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

James suggested at story refinement that we will want to do a quick regression across different devices & browsers, ensuring it’s loaded correctly for font loading.

From Catharine: Consider accessibility tests, ensuring we understand the trade-off between the semantic information offered by `<i>` tags, & the "text splitting" behaviour.

Dev insight: Feeling that unit tests might be sufficient -- though worth discussing if we want to test html structure for presence of `i` and `b` tags and following behaviour (do we have test data with this scenario?).

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1276

**Overall change:** Bump Psammead components and use new fontface

**Code changes:**

- Pulls in all new changes from psammead
- Uses all 6 font faces and new font family

**Testing notes:**  
Part of the ticket tests headlines italic, I have updated `c0000000009o` to have an italic subheadline. It has malformed due to the json output that will be fixed in https://github.com/bbc/simorgh/pull/1395, however the headlines are visible enough to see them.

This ticket also pulls in all psammead components and their changes. The versions used can be found here https://github.com/bbc/simorgh/pull/1394/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2 and The changes to check for can be found in the changelogs of https://github.com/BBC-News/psammead components 
```
Changes from change logs

"@bbc/gel-foundations": Add font family definitions, which include GEL-defined font fallbacks.
"@bbc/psammead-assets": n/a
"@bbc/psammead-brand": Limit banner svg to group 5
"@bbc/psammead-caption": Update to new font face and family
"@bbc/psammead-copyright": Update to new font face and family
"@bbc/psammead-figure": n/a
"@bbc/psammead-headings": Updating colour of headline and subheading to Shadow. Update Subheadings to use Reith Sans Bold. Update to new font face and family.
"@bbc/psammead-image": n/a
"@bbc/psammead-image-placeholder": n/a
"@bbc/psammead-inline-link": Remove top and buttom padding Update colours to reflect new UX designs.
"@bbc/psammead-paragraph": Update appearance to reflect UX designs. Update to new font face and family.
"@bbc/psammead-sitewide-links": Update background colour to Ebon & border colours to Shadow. Constrain content width to Group 5 min width.
"@bbc/psammead-styles": Move font fallback definitions to gel-foundations, add font-face constants.
"@bbc/psammead-visually-hidden-text": n/a

Changes impact

Check new font faces work (fonts load correctly, are used as expected, load correctly etc)
Check brand maxes out at group 5 (1280px)
Check all text uses reith correct as expected in zeplin
Check headline and subheadline colours/weights are correct as expected in zeplin
Check inline links no longer have top and buttom padding and check colours are correct as expected in zeplin
Check Paragraph is using correct colour (C_SHADOW) and are as expected in zeplin
Check footer is background colour to Ebon & border colours to Shadow. Has max width of Group 5 (1280px)
```

I have made an issue to ensure we add an example of italic headlines into psammead https://github.com/bbc/psammead/issues/341

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/lib/globalStyles.js [hunk 1]
```diff
@@ -14,20 +22,12 @@ const GlobalStyle = createGlobalStyle`
...
-    font-style: normal;
-    font-weight: 600;
-    src: url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff2') format('woff2'), url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff') format('woff');
-  }
+  ${F_REITH_SERIF_MEDIUM}
+  ${F_REITH_SERIF_MEDIUM_ITALIC}
+  ${F_REITH_SANS_REGULAR}
+  ${F_REITH_SANS_ITALIC}
+  ${F_REITH_SANS_BOLD}
+  ${F_REITH_SANS_BOLD_ITALIC}
```
Comment (sareh) [inline src/app/lib/globalStyles.js:30]:
```suggestion
```
We should only have these fonts on our News pages:
```
Serif Medium
Serif Medium Italic
Sans Regular
Sans Regular Italic
Sans Bold
Serif Light (only required for the Most read component, which we currently do not have)
```
Source: https://github.com/bbc/simorgh/issues/1311

**Review by sareh — CHANGES_REQUESTED** [review]
Looks great. Just a note about the fonts being pulled in!

File: src/app/lib/globalStyles.js (hunk 1 above)
Comment (sareh) [reply src/app/lib/globalStyles.js:30]: Having checked with Jim regarding this change in fonts, I've now updated issue #1311 & I'll remove this suggestion.

File: src/app/components/Timestamp/index.jsx [hunk 2]
```diff
@@ -2,15 +2,17 @@ import React from 'react';
...
+} from '@bbc/gel-foundations/typography';
 import { C_CLOUD_DARK } from '@bbc/psammead-styles/colours';
-import { FF_NEWS_SANS_REG } from '@bbc/psammead-styles/fonts';
 
 const StyledTimestamp = styled.span`
   ${GEL_BREVIER};
   color: ${C_CLOUD_DARK};
   display: block;
-  font-family: ${FF_NEWS_SANS_REG};
+  font-family: ${GEL_FF_REITH_SANS};
```
Comment (sareh) [inline src/app/components/Timestamp/index.jsx:15]:
```suggestion
  font-family: ${GEL_FF_REITH_SANS};
  font-weight: 500;
```
I believe everywhere you've added a `font-family`, you should have the relevant `font-weight` value used as well.

**Review by sareh — CHANGES_REQUESTED** [review]
Just a missing `font-family` value. Otherwise it looks great.

File: src/app/components/Timestamp/index.jsx (hunk 2 above)
Comment (dr3) [reply src/app/components/Timestamp/index.jsx]: Really? even when its not bold?

Comment (dr3) [reply src/app/components/Timestamp/index.jsx:15]: We currently dont define a value for regular https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.js#L4-L9 so I think we should maintain not defining one here for now

Comment (sareh) [reply src/app/components/Timestamp/index.jsx:15]: Sounds good.  I was under the impression that Regular used the `font-weight` value equivalent of `normal` which is `500`. But that is apparently not the case.  We should separately discuss which font-weight Regular should use with UX.

**Review by sareh — DISMISSED** [review]
👍 Looks great.

**Review by twinlensreflex — DISMISSED** [review]
LGTM 👍

**Review by twinlensreflex — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]
👍 Looks good! Thanks for adding all the changes in the PR description, for greater visibility. 💯

**Review by pjlee11 — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Pull request merged**